wordpress-mobile/AztecEditor-iOS
# Release v1.11.0

This release includes:

- Update project to have iOS 11 has a minimum https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1222
- Fix color styling https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1215
- Allow configuration of default fonts used in TextView https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1221

To test:
- Build and Run https://github.com/wordpress-mobile/WordPress-iOS/pull/12691
- Check that the improvements referenced behave properly.
